accesspoc/dips/models.py

```diff
@@ -297,3 +349,11 @@ class PREMISEvent(models.Model):
 
     def __str__(self):
         return self.uuid
+
+
+class Setting(models.Model):
```

Comment:
I think that this model should be the one doing all the JSON encoding/decoding.

```python
setting = Setting.objects.get(name="setting_name")
setting.value  # This should already be decoded instead of the JSON blob.
```

I suspect for this to be possible you're going to need to implement your own subclass of `Field` or another class in the hierarchy if fields that can ease the implementation. Django seems to have a `JSONField` but only for Postgres. SQLite has some native JSON support but I don't know if it's relevant here.

Maybe this can help? https://github.com/rpkilby/jsonfield2. The sources are pretty small, nice to read: https://github.com/rpkilby/jsonfield2/tree/master/jsonfield.